test/Serializer/SerializerTest.php

```diff
@@ -1,13 +1,28 @@
 <?php
 
-namespace CultuurNet\SearchV3\Test\Serializer;
+namespace CultuurNet\SearchV3\Serializer;
 
-use CultuurNet\SearchV3\Serializer\Serializer;
 use CultuurNet\SearchV3\ValueObjects\Address;
+use CultuurNet\SearchV3\ValueObjects\Audience;
+use CultuurNet\SearchV3\ValueObjects\Collection;
+use CultuurNet\SearchV3\ValueObjects\ContactPoint;
+use CultuurNet\SearchV3\ValueObjects\Event;
+use CultuurNet\SearchV3\ValueObjects\FacetResult;
+use CultuurNet\SearchV3\ValueObjects\FacetResultItem;
+use CultuurNet\SearchV3\ValueObjects\FacetResults;
+use CultuurNet\SearchV3\ValueObjects\GeoPoint;
+use CultuurNet\SearchV3\ValueObjects\MediaObject;
+use CultuurNet\SearchV3\ValueObjects\Organizer;
 use CultuurNet\SearchV3\ValueObjects\PagedCollection;
+use CultuurNet\SearchV3\ValueObjects\Place;
+use CultuurNet\SearchV3\ValueObjects\PriceInfo;
+use CultuurNet\SearchV3\ValueObjects\Term;
+use CultuurNet\SearchV3\ValueObjects\TranslatedAddress;
+use CultuurNet\SearchV3\ValueObjects\TranslatedString;
 use JMS\Serializer\SerializerInterface;
```

Comment:
This doesn't seem to be used?

Reply:
Removed in https://github.com/cultuurnet/search-v3/pull/43/commits/80ed4010a155458a9a71ea3b70413ad78aa714bd